File: LayoutTests/http/tests/loading/resources/post-in-iframe-with-back-navigation-page-2.php
<!DOCTYPE html>
<h1>Page 2</h1>
<span id="submissionTime"><?php print microtime();?></span><br/>
<form action="./post-in-iframe-with-back-navigation-page-3.php" name="form" method="POST">
</form>
<a id="link" href="javascript:document.form.submit();">to page 3</a><br/>
<a id="backLink" href="javascript:history.back();">go back</a>

File: LayoutTests/http/tests/loading/resources/post-in-iframe-with-back-navigation-page-2.py
#!/usr/bin/env python3

import sys
import time

now = time.time()
now = str(now).split('.')
now = '{} {}'.format(now[1], now[0])

sys.stdout.write(
    'Content-Type: text/html\r\n\r\n'
    '<!DOCTYPE html>\n'
    '<h1>Page 2</h1>\n'
    '<span id="submissionTime">0.{}</span><br/>\n'
    '<form action="./post-in-iframe-with-back-navigation-page-3.py" name="form" method="POST">\n'
    '</form>\n'
    '<a id="link" href="javascript:document.form.submit();">to page 3</a><br/>\n'
    '<a id="backLink" href="javascript:history.back();">go back</a>\n'.format(now)
)